internal/statemachine/helper.go

```diff
@@ -756,6 +756,17 @@ func mountFromHost(targetDir, mountpoint string) (mountCmd, umountCmd *exec.Cmd)
 	return mountCmd, umountCmd
 }
 
+// mountNewFS creates a new filesystem and mounts it at the specified location
+func mountNewFS(targetDir, scratchDir, mountpoint string) (mountCmd, umountCmd *exec.Cmd, err error) {
+	tempDir, err := osMkdirTemp(scratchDir, strings.Trim(mountpoint, "/"))
+	if err != nil {
+		return nil, nil, err
+	}
+	mountCmd = execCommand("mount", "--bind", tempDir, filepath.Join(targetDir, mountpoint))
```

Comment:
This is fine, especially if that's what live-build is doing. I suppose it's easier to bind mount to a temporary directory than mounting a tmpfs and having to give and explicit size of it there. I guess that makes sense.